lib/foreman_tasks/cleaner.rb

```diff
@@ -149,21 +182,26 @@ def prepare_filter
       filter_parts.select(&:present?).join(' AND ')
     end
 
-    def start_tracking_progress
+    def start_tracking_progress(name, total = tasks.size)
+      say "About to remove #{total} #{name}(s)"
       if verbose
         @current = 0
-        @total = tasks.size
+        @total = total
         say "#{@current}/#{@total}", false
       end
     end
 
     def report_progress(chunk)
       if verbose
-        @current += chunk.size
+        @current += chunk.count
```

Comment:
When testing, I was seeing

```
About to remove 3 orphaned execution plan(s)
Deleted  orphaned execution plan(s)
```
It seems the `@current` was not set in non-berbose mode. I guess, we should track the `@current` always, and only prevent the actual printing in the `verbose` mode.